practice-two/src/style/layouts/header.css

```diff
@@ -0,0 +1,136 @@
+.main-header {
+  background-image: url("../../assets/images/background-header.png");
+  background-position: center;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 5px 10px;
+}
+
+.navbar__logo {
+  color: var(--color-primary);
+  font-size: var(--font-size-2xlg);
+  font-style: var(--font-weight-primary);
+  font-weight: var(--font-style-default);
+  line-height: var(--line-height-normal);
+  letter-spacing: var(--letter-spacing-xlg);
+}
+
+.navbar__container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.icon-menu {
+  transition: transform 0.3s ease;
+
+  &:hover {
+    transform: scale(1.1);
+  }
+}
+
+.navbar__content {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .navbar {
+    padding: 20px 15px ;
+  }
+  
+
+  .icon-menu {
+    display: none;
```

Comment:
Should set display none for `navbar__container` instead of only display none for image